ipam/store.go

```diff
@@ -17,7 +17,8 @@ package ipam
 
 import (
 //	"database/sql"
-//	"github.com/romana/core/common"
+"log"
+	"github.com/romana/core/common"
 )
 
 type Vm struct {
```

Comment:
Docstring. Also, why is this called 'Vm'? Does it not represent and 'endpoint'? Since we can also deal with Kubernetes, 'Endpoint' might be a better name?